src/SDK/CcExpiration.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace OneSpanSign.Sdk
+{
+	
+	
+	public class CcExpiration
+	{
+		
+		// Fields
+		
+		// Accessors
+		
+    public Nullable<Int32> Month
```

Comment:
When there is no logic in getters/setters I think inlining them should be preferred. like { get; set; }

Reply:
Got it, will be done.

Reply:
Done.